include/openenclave/internal/cert.h

```diff
@@ -134,6 +153,22 @@ oe_result_t oe_cert_verify(
     size_t num_crls,
     oe_verify_cert_error_t* error);
 
+/**
+ * Verify a self-signed certificate's signature
+ *
+ *
+ * @param cert verify this certificate
+ * @param error Optional. Holds the error message if this function failed.
+ *
+ * @return OE_OK verify ok
+ * @return OE_VERIFY_FAILED
+ * @return OE_INVALID_PARAMETER
+ * @return OE_FAILURE
+ */
+oe_result_t oe_verify_self_signed_cert(
```

Comment:
Do we need a separate method? I'm a little uncomfortable with this, since it doesn't actually even validate that the cert is self-signed. I would prefer making `chain` an optional parameter in `oe_cert_verify` and have the impl do the self-signed check instead of the cert chain validation.

Reply:
Very good suggestion.
I replaced it with or_cert_verify as suggested.